packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/overview-identity/overview-identity.html

```diff
@@ -0,0 +1,340 @@
+<div data-ng-if="!$ctrl.editMode">
```

Comment:
**nitpick(non-blocking):** We can get rid of `ng-scope` and `ng-binding` classes.